wolfssl/wolfcrypt/types.h

```diff
@@ -252,7 +252,8 @@ decouple library dependencies with standard string, memory and so on.
          #endif
     #endif
 
-    #if (defined(_MSC_VER) && !defined(WOLFSSL_NOT_WINDOWS_API)) || \
+    #if (defined(_MSC_VER) && (_MSC_VER == 1200)) ||  /* MSVC6 */ \
+        (defined(_MSC_VER) && !defined(WOLFSSL_NOT_WINDOWS_API)) || \
```

Comment:
Is the customer reporting this defining `WOLFSSL_NOT_WINDOWS_API`? If they are building in MSVC 6.0 shouldn't that not be defined and shouldn't the code use this code path?

Reply:
They are defining `WOLFSSL_NOT_WINDOWS_API`, which they need because windows.h is not needed. However, MSVC 6.0 does not have `long long`, but it does support `__int64` and `unsigned __int64`, which are independent of windows.h

So this is specific to the MSVC 6.0 environment.